app/test_split_integer.py

```diff
@@ -2,20 +2,36 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 15
+    number_of_parts = 3
+    result = split_integer(value, number_of_parts)
+    assert sum(result) == value
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    value = 20
+    number_of_parts = 5
+    result = split_integer(value, number_of_parts)
+    assert len(result) == number_of_parts
+    assert min(result) == max(result)
```

Comment:
No need for this check here, you could `assert` the expected output directly, e.g. `assert function(value_1, value2) == [5, 5, 5, 5]`